CMake Generate issues with multi-config and tracy enabled

I'm not 100% sure why but https://github.com/beyond-all-reason/spring/pull/1762 introduced an issue with multi-config generators where the generate step would fail. I guess maybe bumping the CMake version exposed this?

Example output:
```
CMake Error in rts/System/Sound/CMakeLists.txt:
  IMPORTED_LOCATION not set for imported target "vorbis::vorbisfile"
  configuration "Tracy".


CMake Error in rts/System/Sound/CMakeLists.txt:
  IMPORTED_LOCATION not set for imported target "vorbis::vorbis"
  configuration "Tracy".


CMake Error in rts/System/Sound/CMakeLists.txt:
  IMPORTED_LOCATION not set for imported target "Ogg::ogg" configuration
  "Tracy".


CMake Error in rts/builds/legacy/CMakeLists.txt:
  IMPORTED_LOCATION not set for imported target "OpenGL::GL" configuration
  "Tracy".


CMake Error in rts/builds/legacy/CMakeLists.txt:
  IMPORTED_LOCATION not set for imported target "OpenGL::GLU" configuration
  "Tracy".


CMake Error in rts/builds/legacy/CMakeLists.txt:
  IMPORTED_LOCATION not set for imported target "GLEW::GLEW" configuration
  "Tracy".


CMake Error in rts/builds/legacy/CMakeLists.txt:
  IMPORTED_LOCATION not set for imported target "Fontconfig::Fontconfig"
  configuration "Tracy".


CMake Error in rts/CMakeLists.txt:
  IMPORTED_LOCATION not set for imported target "libunwind::libunwind"
  configuration "Tracy".


CMake Error in rts/System/Sound/CMakeLists.txt:
  IMPORTED_LOCATION not set for imported target "vorbis::vorbisfile"
  configuration "Tracy".


CMake Error in rts/System/Sound/CMakeLists.txt:
  IMPORTED_LOCATION not set for imported target "vorbis::vorbis"
  configuration "Tracy".


CMake Error in rts/System/Sound/CMakeLists.txt:
  IMPORTED_LOCATION not set for imported target "Ogg::ogg" configuration
  "Tracy".


CMake Error in rts/CMakeLists.txt:
  IMPORTED_LOCATION not set for imported target "libunwind::libunwind"
  configuration "Tracy".


CMake Error in rts/CMakeLists.txt:
  IMPORTED_LOCATION not set for imported target "libunwind::libunwind"
  configuration "Tracy".
```

Note that this happens on all multi-config generators, but is most visible in MSVC because that uses mult-config by default.

The reason for this appears to be in RmlUI, which adds an additional configuration for multi-config:

https://github.com/mikke89/RmlUi/blob/7a06f27db04fe5d13a5dacc19b2b4544673a4eca/CMake/Dependencies.cmake#L64-L68

I guess since we don't specify imported targets for the Tracy configuration this causes problems? I'm not exactly sure why this appears with the bumped cmkae version.

Regardless, even before the bumped cmake version this had some problems. In multi-config like MSVC, you would get RmlUI profiling only if you selected the "Tracy" build when building! The rest of the engine would have Tracy enabled, but not RmlUI. That's obviously not ideal.

It looks like we can force `RMLUI_TRACY_CONFIGURATION` to off to disable this behavior since imo it's undesirable.

--- a/rts/lib/CMakeLists.txt
+++ b/rts/lib/CMakeLists.txt
@@ -85,6 +85,9 @@ if (TRACY_ENABLE)
 	option(RMLUI_TRACY_PROFILING "Enable RmlUi profiling with Tracy." ON)
 	# We have our own memory profiling, never enable RmlUi one.
 	set(RMLUI_TRACY_MEMORY_PROFILING OFF CACHE BOOL "Enable RmlUi tracy profiling")
+	
+	# Disable the additional RmlUI multiconfig configuration as it only applies to RmlUI
+	set(RMLUI_TRACY_CONFIGURATION OFF CACHE BOOL "" FORCE)
 endif()
 
 # We resolve freetype here to make sure RmlUi find_package picks the same

